tests/TestModels/TestPlace.php

```diff
@@ -11,7 +11,7 @@
 use MatanYadaev\EloquentSpatial\Objects\MultiPolygon;
 use MatanYadaev\EloquentSpatial\Objects\Point;
 use MatanYadaev\EloquentSpatial\Objects\Polygon;
-use MatanYadaev\EloquentSpatial\SpatialBuilder;
+use MatanYadaev\EloquentSpatial\Traits\HasSpatial;
```

Comment:
You need to import `SpatialBuilder` because it is used in line 29.
```suggestion
use MatanYadaev\EloquentSpatial\SpatialBuilder;
use MatanYadaev\EloquentSpatial\Traits\HasSpatial;
```